MISC/SetSuperJumpThisFrame.md

```diff
@@ -8,7 +8,47 @@ ns: MISC
 cs_type(int) void SET_SUPER_JUMP_THIS_FRAME(Player player);
 ```
 
+Allows the player to perform super jumps. This function must be called every frame for it to work.
+It basically OR's a flag for a single frame, allowing the ped to perform a super jump only when the flag is set.
+
+This flag does also get set in other native functions, for instance 
+
+[SET_EXPLOSIVE_AMMO_THIS_FRAME](#_0xA66C71C98D5F2CFB), [SET_FIRE_AMMO_THIS_FRAME](#_0x11879CDD803D30F4) and [SET_EXPLOSIVE_MELEE_THIS_FRAME](#_0xFF1BED81BFDC0FE0) do also set this flag.
+
+You can check if a player is using **Super Jump** by calling [IS_PLAYER_USING_SUPER_JUMP](#_0xC7D2C20C) from the server.
+
+*Actual native game code:*
+
+```
+struct_PedData *__fastcall n_SetSuperJump(int Player)
+{
+  struct_PedData *ped; // rax
+
+  ped = (struct_PedData *)getPlayerPedFromHandle(Player);
+  if ( ped )
+  {
+    // Offsets are commented, pedSpecialFlags gets accessed and modified down below
+    ped = (struct_PedData *)ped->pedInfo;       // *(QWORD*)(ped + 0x1078)
+    ped->pedSpecialFlags |= 0x4000u;                   // *(DWORD*)(ped + 0x150)
+  }
+  return ped;
+}
+```
```

Comment:
This is probably useless information to put in here as most people won't know what it even means

Reply:
Yeah, this was just to address the 0.00001 percent that asked for 'source code', but yeah, most people won't even know the underlying meaning of it, will remove.